### ISSUE TITLE
Decay schedule implementation of keras

### ISSUE THREAD

**Issue by MrWanter** [fchollet/keras#4845]
Hi! I'm new to keras, I have one confuse regarding the learning rate decay schedule of keras implementation, it seems that keras only implement the `1/t` decay schedule, but as proposed in[http://cs231n.github.io/neural-networks-3/#anneal](url), step decay is much more preferable, so I'm confused that why keras didn't implement other decay schedule method, and in practice, for example, I'm going to train a network, how to set the decay prameter?

**Comment by kudkudak** [comment]
You can use `LambdaCallback` to create your own decay schedule

**Comment by MrWanter** [comment]
I see, thank you.
